.gitattributes

```diff
@@ -0,0 +1,11 @@
+/examples export-ignore
```

Comment:
I would just put the big files like `tests`. `.gitattributes` files are hard to keep in sync and I don't think it's worth to bother for a few KBs.

Reply:
Ok, fair. I'll only exclude `tests` and `examples`.

![baobab](https://i.imgur.com/iaJ2A5e.png)